app/views/document_lead_image/_image_list.html.erb

```diff
@@ -0,0 +1,38 @@
+<table>
+  <% @document.images.each_with_index do |image, i| %>
+    <tr>
+      <td class="image-list">
+        <%= image_tag(url_for(image.thumbnail), id: "image-#{i}") %>
+      </td>
+      <td class="image-list" id=<%= "image-#{i}-metadata" %>>
+        <div class="image-list__metadata">
+          <strong><%= t("document_lead_image.index.alt_text") %></strong>: <%= image.alt_text %> <br \>
+          <strong><%= t("document_lead_image.index.caption") %></strong>: <%= image.caption %> <br \>
+          <strong><%= t("document_lead_image.index.credit") %></strong>: <%= image.credit %> <br \>
+          <div class="image-list__lead-image-flag">
+            <% if image == @document.lead_image %>
+              <div class="image-list__lead-image-label">Lead image</div>
+              <div class="image-list__remove-button">
+                <%= form_tag remove_document_lead_image_path(@document), method: :delete do %>
+                <button>Remove</button>
```

Comment:
We could use a component here...

https://govuk-publishing-components.herokuapp.com/component-guide/button

Reply:
We decided to not use a component until the "link-button" (link which behaves like a button) is created.

Reply:
Cool 🇹🇭 .